### PR TITLE
Clear project

### PR DESCRIPTION
**Pull request by Ranisans**
**What kind of change does this PR introduce?**

Add env-cmd, typescript, eslint, material-ui librares.
Remove prop-types (because we write our project on the typescript), lodash, bluma librares.
Add eslint, prettier, typescript config files.
Add type declaration to image file (for import) and to react-helmet.
Remove default project files and config (gatsby-node.js)
Add "Writers" collection to a cms config file.

**Does this PR introduce a breaking change?**

Removed the old working code :)

**What needs to be documented once your changes are merged?**
Fill README.MD

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -5,17 +5,17 @@ please read the [code of conduct](CODE_OF_CONDUCT.md).
 
 ## Setup
```
Comment (aykuli) [inline CONTRIBUTING.md]: я думаю этот файл нам вообще не нужен

File: lambda/hello.js [hunk 2]
```diff
@@ -1,14 +1,14 @@
 // For more info, check https://www.netlify.com/docs/functions/#javascript-lambda-functions
```
Comment (aykuli) [inline lambda/hello.js]: кажется папка lambda нам тоже не нужна

Comment (Ranisans) [reply lambda/hello.js]: File deleted

File: CONTRIBUTING.md (hunk 1 above)
Comment (Ranisans) [reply CONTRIBUTING.md]: File deleted

**Pull request merged**